packages/beacon-node/src/util/graffiti.ts

```diff
@@ -1,8 +1,33 @@
 import {GRAFFITI_SIZE} from "../constants/index.js";
+import {ClientVersion} from "../execution/index.js";
 
 /**
  * Parses a graffiti UTF8 string and returns a 32 bytes buffer right padded with zeros
  */
 export function toGraffitiBuffer(graffiti: string): Buffer {
   return Buffer.concat([Buffer.from(graffiti, "utf8"), Buffer.alloc(GRAFFITI_SIZE, 0)], GRAFFITI_SIZE);
 }
+
+export function getDefaultGraffiti(
+  opts: {private?: boolean},
+  consensusClientVersion?: ClientVersion,
+  executionClientVersion?: ClientVersion
+): string {
+  if (opts.private) {
+    return "";
+  }
+
+  if (consensusClientVersion === undefined) {
+    throw new Error("Consensus client version must be provided if opts.private is set to false");
```

Comment:
is there ever a case where this would not be available? throw a runtime error during block proposal is not great

Reply:
Made the type required, better to find this issue at build time rather than runtime